GW2EIEvtcParser/EIData/ProfHelpers/Revenant/RenegadeHelper.cs

```diff
@@ -63,5 +65,57 @@ internal static bool IsKnownMinionID(int id)
+            {
+                var skill = new SkillModeDescriptor(player, Spec.Revenant, CitadelBombardment);
+                if (log.CombatData.TryGetEffectEventsBySrcWithGUID(player.AgentItem, EffectGUIDs.RenegadeCitadelBombardment1, out IReadOnlyList<EffectEvent> citadelBombardmentHits))
+                {
+                    foreach (EffectEvent effect in citadelBombardment)
+                    {
+                        Point3D playerPosition = player.AgentItem.GetCurrentPosition(log, effect.Time);
+                        if (playerPosition != null)
+                        {
+                            var playerConnector = new PositionConnector(playerPosition);
+                            var positions = new List<Point3D>();
+                            foreach (EffectEvent hitEffect in citadelBombardmentHits.Where(x => x.Time >= effect.Time && x.Time <= effect.Time + 3000))
+                            {
+                                positions.Add(hitEffect.Position);
+
+                                // Shooting Animation
+                                long animationDuration = hitEffect.Time - effect.Time;
+                                (long start, long end) lifespanAnimation = (effect.Time, effect.Time + animationDuration);
+                                var startPoint = new ParametricPoint3D(playerPosition, lifespanAnimation.start);
+                                var endPoint = new ParametricPoint3D(hitEffect.Position, lifespanAnimation.end);
+                                var rotationConnector = new AngleConnector(effect.Rotation.Z);
+                                var shootingArrow = (RectangleDecoration)new RectangleDecoration(15, 100, lifespanAnimation, color, 0.5, new InterpolationConnector(new List<ParametricPoint3D>() { startPoint, endPoint }))
+                                    .UsingRotationConnector(rotationConnector)
+                                    .UsingSkillMode(skill);
+                                replay.Decorations.Add(shootingArrow);
+
+                                // Hit circles
+                                (long, long) lifespanHit = hitEffect.ComputeLifespan(log, 500);
+                                var connector = new PositionConnector(hitEffect.Position);
+                                replay.Decorations.Add(new CircleDecoration(120, lifespanHit, color, 0.5, connector).UsingFilled(false).UsingSkillMode(skill));
+                            }
+                            // AoE Radius and icons
+                            (long, long) lifespan = (effect.Time, effect.Time + 3000);
+                            var centralPoint = Point3D.FindCentralPoint(positions);
+                            var centralConnector = new PositionConnector(centralPoint);
+                            replay.Decorations.Add(new IconDecoration(ParserIcons.EffectCitadelBombardmentPortal, CombatReplaySkillDefaultSizeInPixel, CombatReplaySkillDefaultSizeInWorld, 0.5f, lifespan, playerConnector)
```

Comment:
The icon will overlap with the icon of the player. Moreover, the portal does not follow the player, you can move around during the cast, the portal will remain at the position where the cast is finished